python/setup.cfg

```diff
@@ -0,0 +1,59 @@
+#   LICENSE
+# Copyright (c) 2021 Genome Research Ltd.
+# Author: Cancer Genome Project cgphelp@sanger.ac.uk
+#
+#
+# This file is part of CASM-Smart-Phase.
+#
+# CASM-Smart-Phase is free software: you can redistribute it and/or modify it under
```

Comment:
FYI, embeding the program name in this block of the AGPL-3 preamble is not standard, see the example linked for auto add/checks.

Reply:
Corrected in all LICENSES. Was following previous examples.